src/Expressions/CaseOperatorExpression.php

```diff
@@ -136,7 +136,6 @@ public function addRecursiveExpression(token_list $tokens, int $pointer, bool $n
 
   <<__Override>>
   public function __debugInfo(): dict<string, mixed> {
-    $last_case = C\lastx($this->whenExpressions);
```

Comment:
This code would throw is there were zero whenExpressions. Removing the function call is not 100% safem if this was behavior that we rely upon.